util/authkit/src/index.ts

```diff
@@ -8,6 +8,7 @@ export type LoginOutcome = "forbidden" | {
     username: string;
     displayName: string;
     roles: string[];
+    email: string;
```

Comment:
```suggestion
    email?: string;
```

Optional, as mentioned above